src/components/Match/MatchLog.jsx

```diff
@@ -34,6 +34,14 @@ const generateLog = (match, selectedTypes) => {
         detail: `${entry.key}`,
       })));
     }
+    if (selectedTypes.runes) {
+      log = log.concat((player.runes_log || []).map(entry => ({
+        ...entry,
+        ...player,
+        type: 'runes',
+        detail: `${entry.key}`,
```

Comment:
Yeah, I think it should be picture with a tooltip.  I think the data-tip attribute will create it, @blukai can confirm but there are a few examples already:

https://github.com/odota/ui/blob/1ac73d6c79b0dfd9808706d8b0602fe93cad40a7/src/components/Match/CrossTable.jsx